src/main/groovy/com/adtran/ScalaMultiVersionPlugin.groovy

```diff
@@ -130,6 +132,9 @@ class ScalaMultiVersionPlugin implements Plugin<Project> {
                         startParameter.projectProperties["recursed"] = true
                         startParameter.excludedTaskNames += getRunOnceTasks()
                         tasks = project.gradle.startParameter.taskNames
+                        if (6 <= project.gradle.gradleVersion.split(Pattern.quote('.'))[0].toInteger() ){
```

Comment:
I think this deserves a comment, please. At least a reference to the issue number.

Reply:
```suggestion
                        // workaround name clash issue in Gradle 6+. See #27.
                        if (6 <= project.gradle.gradleVersion.split(Pattern.quote('.'))[0].toInteger() ){
```

Reply:
cool, good call